projects/cds/src/profiletag/tracking/tracking.module.ts

```diff
@@ -0,0 +1,16 @@
+import { APP_INITIALIZER, NgModule } from '@angular/core';
+import { TrackingService } from './tracking.service';
+
+@NgModule({
+  providers: [
+    {
+      multi: true,
+      provide: APP_INITIALIZER,
+      useFactory: (trackingService: TrackingService) => () => {
```

Comment:
It seems that this is not supported by the API extractor bot (see https://github.com/SAP/spartacus/pull/8226/checks?check_run_id=1219219260#step:3:223)